mooncake-store/src/master.cpp

```diff
@@ -69,45 +71,45 @@ DEFINE_string(cluster_id, mooncake::DEFAULT_CLUSTER_ID,
 
 DEFINE_string(memory_allocator, "offset",
               "Memory allocator for global segments, cachelib | offset");
+void InitMasterConf(const mooncake::DefaultConfig& default_config, 
+                    mooncake::MasterConfig& master_config) {
+    // Initialize the master service configuration from the default config
+    default_config.GetBool("enable_gc", &master_config.enable_gc, false);
+    default_config.GetBool("enable_metric_reporting",
+                            &master_config.enable_metric_reporting, true);
+    default_config.GetUInt32("metrics_port", &master_config.metrics_port, 9003);
```

Comment:
Currently, each parameter's default value has to be specified twice.
To improve consistency and reduce redundancy, would it make sense to first check info.is_default, and only if it's false, read the value from the file?
Other approaches that achieve a similar effect are also totally fine.

Reply:
This workflow looks good overall. However, the current implementation requires each parameter’s default value to be specified twice.

Reply:
Leave the command default value and if the config file parse error, the parameter’s value will be 0 or '', then set the default value from command ?